src/components/Tokens/index.tsx

```diff
@@ -82,24 +81,24 @@ const Tokens = ({ variant = 'optionSelector', callbackFunction, showBalances = t
       {
         token: RELAY_CHAIN_NATIVE_TOKEN,
         type: TokenType.RelayChainNative,
-        balance: displayMonetaryAmount(collateralTokenBalance),
-        transferableBalance: displayMonetaryAmount(collateralTokenTransferableBalance),
+        balance: collateralTokenBalance.toHuman(5),
```

Comment:
These are the critical changes - these values are being used for transactions/validation, so can't be returned as display amounts